projects/kendo-ui-buttons-ngcc/src/app/app.component.ts

```diff
@@ -7,6 +7,22 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'kendo-ui-misc-ngcc';
+  chips = ['Chips', 'Snack', 'Smack'];
```

Comment:
There seems to be a bunch of kendo up changes that perhaps are not supposed to be in this commit?

Reply:
Without these changes, building these kendo-ui projects fails (probably due to the switch from jit to aot compilation), so I wanted to include them in the commit that makes them necessary.